### PR TITLE
Customizable URI ID used for offline signature

### PR DESCRIPTION
**Pull request by kober32**
Resolves issue #78

### CODE REVIEW

File: WultraMobileTokenSDK/Operations/Service/WMTOperationsImpl.swift [hunk 1]
```diff
@@ -290,10 +293,26 @@ class WMTOperationsImpl: WMTOperations {
     ///                 This completion is always called on the main thread.
     /// - Returns: Operation object for its state observation.
     func authorize(qrOperation: WMTQROperation, authentication: PowerAuthAuthentication, completion: @escaping (Result<String, WMTError>) -> Void) -> Operation {
+        return authorize(qrOperation: qrOperation, uriId: "/operation/authorize/offline", authentication: authentication, completion: completion)
```
Comment (hvge) [inline WultraMobileTokenSDK/Operations/Service/WMTOperationsImpl.swift]: Move this to a protocol extension.

File: WultraMobileTokenSDK/Operations/WMTOperations.swift [hunk 2]
```diff
@@ -94,6 +97,22 @@ public protocol WMTOperations: AnyObject {
     @discardableResult
     func authorize(qrOperation: WMTQROperation, authentication: PowerAuthAuthentication, completion: @escaping(Result<String, WMTError>) -> Void) -> Operation
```
Comment (hvge) [inline WultraMobileTokenSDK/Operations/WMTOperations.swift]: Move this to a protocol extension:
```swift
public extension WMTOperations {
    @discardableResult
    func authorize(qrOperation: WMTQROperation, authentication: PowerAuthAuthentication, completion: @escaping(Result<String, WMTError>) -> Void) -> Operation {
        return authorize(qrOperation: qrOperation, uriId: "/operation/authorize/offline", authentication: authentication, completion: completion)
    }
}
```

**Review by hvge — CHANGES_REQUESTED** [review]

File: WultraMobileTokenSDK/Operations/WMTOperations.swift (hunk 2 above)
Comment (kober32) [reply WultraMobileTokenSDK/Operations/WMTOperations.swift]: Good point - moved

**Review by hvge — APPROVED** [review]
Looks OK now

**Pull request merged**